testem.json

```diff
@@ -1,11 +1,21 @@
 {
   "framework": "mocha",
-  "test_page": "tests/index.html?hidepassed&coverage",
+  "test_page": "tests/index.html?hidepassed",
   "disable_watching": true,
   "launch_in_ci": [
-	"Firefox"
+    "Firefox"
   ],
   "launch_in_dev": [
-	"Chrome"
-  ]
+    "Firefox"
```

Comment:
Covered from what perspective?  The screenshots for this are going to be taken using Slimer currently and test execution can occur against any of the major browsers - our standard for addons currently is to use Firefox (we may swap to Chrome once ember-cli-code-coverage is ready)